### ISSUE TITLE
Todos

### ISSUE THREAD

**Issue by AnHaechan** [AnHaechan/ai-compilers-study-material#2]
- [ ] Add CUDA Tile IR, CuTile Python
- [ ] Expand LLM-based kernel generatiom
   - [ ] The benchmark: KernelBench
   - [ ] Meta's KernelEvolve: LLM + Evolutionary algo.
   - [ ] RightNow's [Forge](https://www.rightnowai.co/blog/llms-write-gpu-kernels-beat-torch-compile): parallel coder-judge pairs

**Comment by AnHaechan** [comment]
- [ ] Add explanation on FlexAttention and FlashInfer & Mark FlashInfer as important
- [ ] Based on the FlashInfer paper, add more explanation on the attention literature ('23-'25)
  - PagedAttention, RadixAttention (for prefix caching)
  - speculative decoding, parallel decoding, TreeAttention
  - BPT, FlashDecoding, RingAttention, LeanAttention (chunked / composable attentions)
  - RoPE, specialized masks on logits, MLA, GQA, sliding window attention
  - LinearAttention
